bcda/cli.go

```diff
@@ -45,7 +46,59 @@ type cclfFileMetadata struct {
 	filePath  string
 }
 
-func importCCLF8(fileMetadata cclfFileMetadata) error {
+type cclfFileValidator struct {
+	totalRecordCount int
+	maxRecordLength  int
+}
+
+func importCCLF0(fileMetadata cclfFileMetadata) (map[string]cclfFileValidator, error) {
+	var validator map[string]cclfFileValidator
+	if fileMetadata.filePath == "" {
+		return nil, errors.New("file path (--file) must be provided")
```

Comment:
The `--file` reference here is from when CCLF8/9 were imported directly with CLI commands, instead of the first step of looking at the whole directory that we have now. So that flag can be removed from the message

Reply:
oooh good catch!